web/src/features/charts/FuturePrice.tsx

```diff
@@ -115,27 +117,32 @@ export function FuturePrice({ futurePrice }: { futurePrice: FuturePriceData | nu
                                   granularity
                                 )}
                               />
-                            )}
-                          </div>
-                        )}
-                        <div
-                          style={{
-                            width: `${100 - negativePercentage}%`,
-                          }}
-                          data-test-id="positive-price"
-                        >
-                          {price > 0 && (
-                            <PriceBar
-                              price={price}
-                              maxPrice={maxPriceTotal}
-                              color={getColor(
-                                price,
-                                maxPriceFuture,
-                                minPriceFuture,
-                                date,
-                                granularity
+                            </div>
+                          )}
+                          {price >= 0 && (
+                            <div
+                              data-test-id="positive-price"
+                              style={{
+                                width: `calc(calc(calc(100% - 12px) * ${
+                                  (100 - negativePercentage) / 100
+                                }) + 12px)`,
+                              }}
```

Comment:
```suggestion
                              style={{
                                width: `calc(100% * ${(100 - negativePercentage) / 100} + 12px - 12px * ${(100 - negativePercentage) / 100})`,
                              }}
```
same here. You'll probably have to run the formatting afterwards though 😅